src/foam/nanos/crunch/Capability.js

```diff
@@ -95,6 +102,14 @@ foam.CLASS({
       class: 'StringArray',
       documentation: `List of permissions granted by this capability`
     },
+    {
+      name: 'price',
```

Comment:
Is this supposed to represent a monetary value? If so, we need to also store a reference to the currency the price is in.

Reply:
Yes.

Reply:
@KernelDeimos you'll need an additional property to hold the 'currency' value.